chain/indexer.go

```diff
@@ -389,9 +360,6 @@ func (t *TipSetIndexer) TipSet(ctx context.Context, ts *types.TipSet) error {
 		if res.Error != nil {
 			llt.Errorw("task returned with error", "error", res.Error.Error())
 			// tell all the processors to close their connections to the lens, they can reopen when needed
-			if err := t.closeProcessors(); err != nil {
```

Comment:
task no longer use opener pattern so they do not need to be closed. The TipSetIndexer **does** still have a close method to deal with the persistence slot, which is unchanged.